README.rst

```diff
@@ -27,7 +27,12 @@ This is easily achieved by downloading
 `the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
 Installing from PyPI
+<<<<<<< HEAD
```

Comment:
Looks like you had a merge conflict here; please resolve and keep an eye out for them. An easy way to check for them after the fact is to grep for `<<<<` or `>>>>`